example-get-started/code/src/evaluate.py

```diff
@@ -42,7 +42,7 @@
         {
             "prc": [
                 {"precision": p, "recall": r, "threshold": t}
-                for p, r, t in zip(precision, recall, prc_thresholds)
+                for p, r, t in list(zip(precision, recall, prc_thresholds))[::2]
```

Comment:
let's put a comment that we are taking half of the points to make it faster for now?